packages/ui/src/kit/VTable/VTableHeadCell/VTableHeadCell.vue

```diff
@@ -1,21 +1,97 @@
 <template>
   <th :class="vTableHeadCellCssClass">
-    <slot />
+    <a
```

Comment:
if we are not using `<a>` for other cases, we can use `v-if=isSortable` for `<a>` tag here